docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+
+# Getting started
+
+## Repository structure
+
+The repository contains multiple separate components.
+The API server and web client are split into their own *npm* modules.
+Dockerfiles for development and production are also provided.
+
+Both the client and server are continuously tested using [Travis CI][travis].
+The Travis configuration `.travis.yml` is shared and located at the root.
+In the root there is a repo-wide `.editorconfig` file, set up [EditorConfig][editorconfig]
+to keep the code style consistent (the linter should pick up most errors though).
+
+## Client and server
+
+All JavaScript is written in modern ES6 and compiled using [Babel][babel].
+This allows using the same set of features both on the frontend and backend.
+The code is linted using [ESLint][eslint] with the [Airbnb][gh-airbnb] style
+guide with the exception of not using semicolons.
+Both Babel `.babelrc` and ESLint `.eslintrc.json` configuration files are duplicated
+in the *server/* and *client/* directories.
+
+NPM scripts for development for server and client:
+
+| Command | Description |
+| --- | --- |
+| `lint` | Verify code style, use `npm run lint -- --fix` to fix mistakes automatically |
+
+Rarely manually needed NPM scripts for server and client:
+
+| Command | Description |
+| --- | --- |
+| `lint:ci` | Lint like the CI does, checking the line ending format. This will likely fail on Windows as Git likes to check out `\r\n` line endings. |
+
+## Client
```

Comment:
As there is already second level header `Client and server`, should this and the `Server` below be third level headers?

Reply:
I don't really know what the cleanest way to format this is, there should be a section with general things shared between the client and the server, and ones dedicated to each. I kinda feel like they belong to the same heading level but might be wrong.

@HenrikHelenius opinions? You kept the Google Docs pretty clean